rpc/rpc-wrappers.go

```diff
@@ -878,6 +880,26 @@ func NewStorageAuthorityServer(rpc Server, impl core.StorageAuthority) error {
 		return
 	})
 
+	rpc.Handle(MethodGetAuthorizationsByDomain, func(req []byte) (response []byte, err error) {
+		ident := core.AcmeIdentifier{}
+		err = json.Unmarshal(req, &ident)
+		if err != nil {
+			return nil, err
+		}
+		authz, err := impl.GetAuthorizationsByDomain(ident)
+		if err != nil {
+			return
```

Comment:
Actually, looking at local style it seems like naked returns are the thing, and this function is short enough. So it's line 887 above that should be made consistent.